test/support/utils.h

```diff
@@ -552,8 +551,7 @@ class Wrapper
     Wrapper&
     operator=(Wrapper&& input)
     {
-        my_field = input.my_field;
-        input.my_field = nullptr;
+        my_field = ::std::move(input.my_field);
```

Comment:
https://en.cppreference.com/w/cpp/memory/shared_ptr/operator%3D
(2) ```shared_ptr& operator=( shared_ptr&& r ) noexcept;```
"Move-assigns a shared_ptr from r. After the assignment, *this contains a copy of the previous state of r, and r is empty. Equivalent to shared_ptr<T>(std::move(r)).swap(*this)."